CSharpMinifier/RoslynMinifier.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CSharpMinifier.Rewriters;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+namespace CSharpMinifier
+{
+    public class RoslynMinifier : IMinifier
+    {
+        public SyntaxTree SyntaxTree { get; private set; }
```

Comment:
It makes sense to remove this property and pass `SyntaxTree` to the method via arguments instead. Advantage: getting rid of yet another side effect.